lib/npm.js

```diff
@@ -69,7 +70,14 @@ function getTarball (name, filename) {
   })
 }
 
+function * getDistTags (name) {
+  let opts = {timeout: config.timeout, json: true}
+  let rsp = yield got(url.resolve(config.uplink.href, '/-/package/' + name.replace(/\//, '%2F') + '/dist-tags'), opts)
```

Comment:
@dickeyxxx could you `encodeURIComponent` just in case name has some other problematic characters?